src/data/users.js

```diff
@@ -1,680 +1,680 @@
-// Do not edit this file
+// // Do not edit this file
```

Comment:
I'd say to probably either remove these files or just remove all references to them (probably the former) -- don't really wanna push commented code up to master